Story/ConcertEvent[MEM].cs

```diff
@@ -0,0 +1,136 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class Concert
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new();
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+        DoAll();
+        Core.SetOptions(false);
+    }
+
+    public void DoAll()
+    {
+        if (!Core.IsMember)
+        {
+            Core.Logger("Concert Event Is Member Only. Skipping this Script");
+            return;
+        }
+
+        Story.PreLoad(this);
+
+        Vertigo();
+        Darkness();
+        Feardeath();
+    }
+
+    public void Vertigo()
+    {
+        if (Core.isCompletedBefore(1024))
+            return;
+
+        //Up to the Highest Heights 1020
+        Story.MapItemQuest(1020, "vertigo", 390);
+
+        //High-sterical 1021
+        if (!Story.QuestProgression(1021))
```

Comment:
Story.Progression is fine keep it. but why didn't you use 
Story.KillQuest(1021, "vertigo", new[] { "Fear Muncher", "Banished Banshee" });
?